Classes/Systems/GithubClient.swift

```diff
@@ -164,4 +199,8 @@ struct GithubClient {
         })
     }
 
+    static func url(baseURL: String = "https://github.com/", path: String) -> URL? {
```

Comment:
So most client-independent URLs have `https://github.com/` as their baseURL, but some [have others](https://github.com/rnystrom/GitHawk/pull/1549/files#diff-9f0bae268086ea5dfc986d1a77f28abaR114) (`https://status.github.com/` in this case). Would it be overkill to put those variations in an enum?

Reply:
Yeah - currently we are using [`https://status.github.com/`](https://github.com/rnystrom/GitHawk/pull/1549/files#diff-9f0bae268086ea5dfc986d1a77f28abaR114) and [`https://api.github.com/`](https://github.com/rnystrom/GitHawk/pull/1549/files#diff-2af355000c118286c09e081675a7b959R36).